packages/expo-sqlite/src/next/NativeStatement.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Result of a `runAsync` call.
+ */
+export interface RunResult {
+  /**
+   * The last inserted row ID.
+   */
+  lastInsertRowid: number;
+
+  /**
+   * The number of rows affected.
+   */
+  changes: number;
+}
+
+/**
+ * Bind parameters to the prepared statement.
+ * You can either pass the parameters in the following forms:
+ *
+ * @example
+ * - Variadic arguments for unnamed parameters.
+ * ```ts
+ * const statement = await db.prepareAsync('SELECT * FROM test WHERE value = ? AND intValue = ?');
+ * await statement.getAsync('test1', 789);
+ * ```
+ *
+ * @example
+ * - A single array for unnamed parameters.
+ * ```ts
+ * const statement = await db.prepareAsync('SELECT * FROM test WHERE value = ? AND intValue = ?');
+ * await statement.getAsync(['test1', 789]);
+ * ```
+ *
+ * @example
+ * - A single object for [named parameters](https://www.sqlite.org/lang_expr.html)
+ *
+ *   Through we support multiple named parameter forms like `:VVV`, `@VVV`, and `$VVV`. We recommend using `$VVV` because JavaScript allows using `$` in identifiers without escaping.
```

Comment:
```suggestion
 * We support multiple named parameter forms such as `:VVV`, `@VVV`, and `$VVV`. We recommend using `$VVV` because JavaScript allows using `$` in identifiers without escaping.
```